pkg/serverless/trace/inferredspan/span_enrichment.go

```diff
@@ -162,6 +162,33 @@ func (inferredSpan *InferredSpan) EnrichInferredSpanWithSQSEvent(eventPayload ev
 	}
 }
 
+// EnrichInferredSpanWithDynamoDBEvent uses the parsed event
+// payload to enrich the current inferred span. It applies a
+// specific set of data to the span expected from a DynamoDB event.
+func (inferredSpan *InferredSpan) EnrichInferredSpanWithDynamoDBEvent(eventPayload events.DynamoDBEvent) {
+	eventRecord := eventPayload.Records[0]
```

Comment:
Might be a dumb question but are we guaranteed that `Records` is always set, and is always >= 1? Just wondering if there can be some weird AWS case where this isn't true.

Reply:
Yes it is always an array 
```
type DynamoDBEvent struct {
	Records []DynamoDBEventRecord `json:"Records"`
}
```